test_vm/vendor/cookbooks/dbfit_test/recipes/mysql.rb

```diff
@@ -23,14 +23,18 @@
 end
 
 users.each do |username, password|
-  mysql_database_user username do
+  mysql_database_user "create #{username}" do
     connection mysql_connection_info
+    username username
     password password
     action :create
   end
 
   %w{localhost 127.0.0.1}.each do |hostname|
-    mysql_database_user username do
+    mysql_database_user "grans on dbfit.* to #{username}@#{hostname}" do
+      connection mysql_connection_info
```

Comment:
Yes, it's a typo (thanks for spotting it!). I just pushed a fix-up. It grants privileges for the test users to the test database. There are some warnings when having several `mysql_database_user <name>` with the same name (as it leads to `resource cloning`).